src/main.rs

```diff
@@ -261,6 +315,22 @@ impl From<CheckRelease> for cargo_semver_checks::Check {
             check.with_release_type(release_type);
         }
 
+        if value.all_features {
+            check.with_all_features();
+        } else if value.default_features {
+            check.with_default_features();
+        } else if value.only_explicit_features {
+            check.with_only_explicit_features();
+        } else {
+            check.with_heuristically_included_features();
+        }
+        let mut mutual_features = value.features;
+        let mut current_features = value.current_features;
+        let mut baseline_features = value.baseline_features;
+        current_features.append(&mut mutual_features.clone());
+        baseline_features.append(&mut mutual_features);
```

Comment:
`.append()` taking a `&mut` seems very strange. What's the reasoning for choosing it, versus taking the argument by value or as read-only ref?

Reply:
I agree, but don't see a method that makes this more readable.